devito/tools/abc.py

```diff
@@ -130,7 +130,11 @@ def __init__(self, a, b, c=4):
             * `a._rebuild(c=5) -> x(3, 5, 5)`
             * `a._rebuild(1, c=7) -> x(1, 5, 7)`
         """
-        args += tuple(getattr(self, i) for i in self.__rargs__[len(args):])
+        for i in self.__rargs__[len(args):]:
+            if i.startswith('*'):
```

Comment:
how do you end up here?

Reply:
the need to use something like `*indices` inside `__rargs__ = (..., '*indices')` for reconstruction of objects with variadic args (e.g., `FIndexed`)